src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -513,6 +524,74 @@ def getThresholds(self):
 
         return
 
+    def _getJobSubmitCondition(self, jobPrio, siteName, jobType):
+        """
+        returns the string describing whether a job is ready to be submitted or the reason can't be submitted
+        Only jobs with "JobSubmitReady" return value will be added to submit job.
+        Other return values will indicate the reason jobs cannot be submitted.
+        i.e. "NoPendingSlot"  - pending slot is full with pending job
+        """
+        try:
+            totalPendingSlots = self.currentRcThresholds[siteName]["total_pending_slots"]
+            totalPendingJobs = self.currentRcThresholds[siteName]["total_pending_jobs"]
+            totalRunningSlots = self.currentRcThresholds[siteName]["total_running_slots"]
+            totalRunningJobs = self.currentRcThresholds[siteName]["total_running_jobs"]
+            highestPriorityInJobs = self.currentRcThresholds[siteName]['wf_highest_priority']
+
+            taskPendingSlots = self.currentRcThresholds[siteName]['thresholds'][jobType]["pending_slots"]
+            taskPendingJobs = self.currentRcThresholds[siteName]['thresholds'][jobType]["task_pending_jobs"]
+            taskRunningSlots = self.currentRcThresholds[siteName]['thresholds'][jobType]["max_slots"]
+            taskRunningJobs = self.currentRcThresholds[siteName]['thresholds'][jobType]["task_running_jobs"]
+
+            totalPendingJobs = self.currentRcThresholds[siteName]["total_pending_jobs"]
```

Comment:
I failed to see any difference :-) Maybe next time make a new commit only to make it clear what changed (and merge after you get the green light).

Anyways, isn't this line L546 == L536? And L550 == L542?

Reply:
Ah I need to change my glasses.  The difference was before it was total_pending_slot. Now it is total_pending_jobs